### ISSUE TITLE
No .mat file created using export function, no errors aswell

### ISSUE THREAD

**Issue by FlorentTKS** [danielhrisca/asammdf#800]
# Python version
_Please run the following snippet and write the output here_

('python=3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 '
 'bit (AMD64)]')
'os=Windows-10-10.0.19045-SP0'
'numpy=1.23.1'
ldf is not supported
xls is not supported
xlsx is not supported
yaml is not supported
'asammdf=7.1.1'

# Code 

from asammdf import MDF
from pathlib import Path

# File path UI
import tkinter as tk
from tkinter import filedialog

# set variables
mdf_extension = ".MF4"
input_folder = "input"
output_folder = "output"

# load MDF/DBC files from input folder
path = Path(__file__).parent.absolute()

# Prompt the user interface for mf4 data selections
root = tk.Tk()
root.withdraw()

# get the path of the mf4 data
mf4_path = tuple(input("MF4_File_Path_Apaix").split())

# Prompt the user interface for dbc selections
root = tk.Tk()
root.withdraw()
# get the path of the dbcs
dbc_path = filedialog.askopenfilenames(parent=root, title='Choose DBC file(s)', filetypes=[("dbc files", ".dbc")])

# handler of cancel or nothing selected
if not dbc_path:
    exit()

# transfer the path here
logfiles = list(mf4_path)
dbc_files = {"CAN": [(dbc, 0) for dbc in dbc_path]}

# concatenate MDF files from input folder and export as CSV incl. timestamps (localized time)
mdf = MDF.concatenate(logfiles)

# DBC convert the unfiltered MDF + save & export resampled data
mdf_scaled = mdf.extract_bus_logging(dbc_files)
mdf_scaled.export(fmt='mat', filename=Path('C:\Temp\APAIX\MDF4\matlab.mat'))

  ## MDF version
  _please write here the file version (you can run ``print(MDF(file).version)``)

  ## Code snippet
  no error
  
  ## Traceback
  no traceback
  
# Description
   Hi, I am trying to convert CAN data in MF4 from a CSS logger to a matfile. Currently it works fine in CSV but the files are too heavy and it will be easier to load it to our data analysis SW in .mat.
Unfortunately, the export function doesn't output any mat file.
I made sure to have scipy installed and checked the other issues on gitHub.

_Please describe the issue here._

**Comment by danielhrisca** [comment]
Do you have `hdf5storage` installed?

**Comment by FlorentTKS** [comment]
yes I think so, I am quite new to Python so i may have forget something, here is what I see when trying to install it via the pip command:

![image](https://user-images.githubusercontent.com/120581985/207657171-2aa59e7f-02f2-446c-a0da-b40ce6d4d387.png)

**Comment by Brezenabsalzer** [comment]
I can confirm that no mat file is created. hdf5storage is installed and hdf5 export works fine.

**Comment by danielhrisca** [comment]
I think the problem is that the hdf5storage library is very slow at writing the file.

**Comment by danielhrisca** [comment]
@Brezenabsalzer @FlorentTKS please try out the development branch

**Comment by MatinF** [comment]
This may be a missing `scipy` installation as well

**Comment by danielhrisca** [comment]
ping @Brezenabsalzer @FlorentTKS

**Comment by tsindel** [comment]
@danielhrisca what is the solution here? I still have the same problem. See the Conda dependencies below:

dependencies:
  - pypy=7.3.15
  - scipy==1.12.0
  - hdf5storage==0.1.19
  - chardet==5.2.0
  - python=3.9.18
  - pip
  - pip:
    - asammdf==7.4.1
    - faust-cchardet==2.1.19
    - isal==1.6.0
